### PR TITLE
REX-5772| "No-funding" attribute to be sent to Scholar One if author claims no funding received

### PR DESCRIPTION
**Pull request by SongRaein**

### CODE REVIEW

File: ScrumUpdate.txt [hunk 1]
```diff
@@ -16,4 +16,6 @@ PArticipated in Internal discussion | Sprint stories clarifications
 Paricipate in Brainstorming session on CMDB Part 2
 Revisit CMDB KTs and refer Documents
 Study ATF
-Study Robot framework (Community work)
\ No newline at end of file
+Study Robot framework (Community work)
+
+Changed!!
```
Comment (lhansamali) [inline ScrumUpdate.txt:21]: Review-> change

**Review by lhansamali — CHANGES_REQUESTED** [review]
Changes

File: ScrumUpdate.txt (hunk 1 above)
Comment (SongRaein) [reply ScrumUpdate.txt:21]: Changed. Please review

**Review by lhansamali — APPROVED** [review]
Good to Go

**Pull request merged**